### ISSUE TITLE
Build Process Video

### ISSUE THREAD

**Issue by cynthiachen28** [amosproj/amos2022ss05-find-my-hearing-aid#85]
## User story
1. As a user
2. I want / need a video shows how to master the build process, steps include Clone, Build, Test, Deploy (optional), and Run. 
3. So that I can know how the project is build.

## Acceptance criteria
* A video includes all four required steps in the build process is created.
* The video is named and uploaded to *sprint-06/{user|design|build}-documentation.pdf*

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by codingwithelijah** [comment]
@cynthiachen28: Thanks again for creating the issue! Did I get it correctly that @Chippelius takes care of this issue?

**Comment by cynthiachen28** [comment]
@Chippelius Thanks and please start with the android version first. I would bring the answer regarding the iOS version later.

**Comment by Chippelius** [comment]
I created and uploaded the build process video. It can be found at https://github.com/amosproj/amos2022ss05-find-my-hearing-aid/blob/main/Deliverables/sprint-05/build-process-video.mp4 
Due to its large size and being a binary file, it has been uploaded via git lfs (see https://docs.github.com/articles/versioning-large-files/ for further information).
